nathan-shulkin.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html lang="en">
+<link rel="stylesheet" href="nathan-shulkin.css">
+  <head>
+    <title>CSS Timeline</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <h1>Brief History of CSS</h1>
+    <ul>
+      <li><div id="green"><div class="chrono">1994</div>: The first work on style sheets started at CERN.</div></li>
```

Comment:
There isn't a need to use multiple `div`s here. Remember, you can apply both classes and ids to s given element - we don't need to create new elements every time we want to apply a class or id.